gallery/psget/psget_cmdlets_troubleshooting.md

```diff
@@ -1,24 +0,0 @@
----
-ms.date:  06/12/2017
-contributor:  manikb
-ms.topic:  reference
-keywords:  gallery,powershell,cmdlet,psget
-title:  psget_cmdlets_troubleshooting
```

Comment:
We may not want to delete this article. But we may want to fold this into another article. Restore it for now so we can talk to Keith about it.